Project.toml

```diff
@@ -1,17 +1,18 @@
 name = "Intervals"
 uuid = "d8418881-c3e1-53bb-8760-2df7ec849ed5"
+license = "MIT"
+desc = "Non-iterable ranges"
```

Comment:
Where are these descriptions used? I can't seem to find anything on Google about them nor in the `Pkg.jl` [documentation](https://julialang.github.io/Pkg.jl/dev/toml-files/)

Reply:
I think it's something that Pkg.jl developers talked about using to allow you to list packages in a registry including the description, but then never implemented. Didn't realize Pkg.jl just silently ignores it.